zonemgr/zone_file.go

```diff
@@ -0,0 +1,198 @@
+package zonemgr
+
+import (
```

Comment:
I think you have some rebase issue. The commit contains a commit message if an old commit.

Reply:
also the first commit is a commit that was merged

Reply:
seems the new code was amended to a previous unrelated merged commit (manifests: Add yaml separator)